Libs/DICOM/Core/ctkDICOMItem.h

```diff
@@ -223,6 +223,7 @@ class CTK_DICOM_CORE_EXPORT ctkDICOMItem
     /// \brief Nicely formatted (group,element) version of a tag
     ///
     static QString TagKey( const DcmTag& tag );
+    static QString TagKeyStripped( const DcmTag& tag );
```

Comment:
Is this method required ? Should TagKey be systematically stripped ?

Reply:
TagKey returns the tag in parentheses. TagKeyStripped does not have the parentheses. I don't think the original one could be removed. However, maybe the name of the new function is not good enough. If you have a better suggestion let me know.

Reply:
What about `TagKeyWithoutParenthesis` ? Or may a static function called `StipParenthesis` could be added ? And the same effect would be achieved doing `StipParenthesis(TagKey(tag))`

Reply:
How about `TagKeyUnpunctuated` or `TagKeyWitoutPunctuation` and `unpunctuateTagKey`?

Reply:
`TagKeyWithoutPunctuation` :+1:

Reply:
I like TagKeyUnpunctuated and TagKeyWithoutPunctuation. There is still a comma between the numbers. Is it punctuation? Because if it is then this name too would be somewhat misleading.
The parentheses version sounds a bit strange to me.

Reply:
I would also be fine keeping `TagKeyStripped`, let's just make sure to add a docstring explaining what the function is doing.

Reply:
Oh, if it only changes removes the parentheses and not the comma then I'd change my vote to `TagKeyWithoutParenthesis` but am also fine with `TagKeyStripped` with documentation.


Just for reference, in [similar javascript code](https://github.com/dcmjs-org/dcmjs/blob/master/src/DicomMetaDictionary.js#L6-L21) we use `punctuateTag` and `unpunctuateTag`.  to go between (XXXX,XXXX), used in the dcmtk-derived data dictionary and XXXXXXXX used in DICOM JSON Model.  My current thinking is that these are close to the standard which uses "tag" made of group and element, but there's no reason to change the usage in CTK at this point.

Reply:
I updated the PR keeping TagStripped but with explanation. I can still change it to TagKeyWithoutParenthesis if it's better for everyone.